docs/site/linting.md

```diff
@@ -7,6 +7,47 @@ description: Powered by clj-kondo, Calva comes with state-of-the-art Clojure and
 
 Calva does no linting, yet with Calva you get excellent linting. That is because Calva uses [clojure-lsp](https://github.com/clojure-lsp/clojure-lsp), which provides linting powered by clj-kondo.
 
+## Get Started
+clj-kondo comes with its set of rules, but the configuration can be customized with a configuration file. The custom configuration must be placed at the root of your project under `.clj-kondo/config.edn`. The `.clj-kondo` folder may already exist if you started calva before, it is safe to create it manually if it doesn't.
```

Comment:
> The custom configuration must be placed ...

I think it can go in some different places, as described here: https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#introduction

Reply:
@PEZ Any chances you know about some of them? I don't know if you get all of them since we have the LSP in between. For instance `command line --config file or EDN arguments` wouldn't apply.

Do you think it's okay to point to the documentation and let the user make the decisions? Or should we somehow add a bit of context and maybe only list the one that apply to using kondo with calva?

Reply:
I think it's OK to point to the documentation. 
```suggestion
clj-kondo comes with great default rules, and the configuration [can be customized](https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md). One of your options for the configuration file is to  placed a `config.edn` file in `.clj-kondo/` at the root of your project. This folder may or may not already exist. It is safe to create it manually if it doesn't.
```